src/Tribe/Views/V2/Hooks.php

```diff
@@ -88,6 +91,7 @@ protected function add_filters() {
 		add_filter( 'tribe_general_settings_tab_fields', [ $this, 'filter_general_settings_tab_live_update' ], 20 );
 		add_filter( 'tribe_events_rewrite_i18n_slugs_raw', [ $this, 'filter_rewrite_i18n_slugs_raw' ], 50, 2 );
 		add_filter( 'tribe_get_event_after', [ $this, 'filter_events_properties' ] );
+		add_filter( 'tribe_events_views_v2_should_cache_html', [ $this, 'filter_suspend_cache_from_admin_bar' ] );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: d1a0453174cefbcfc5288234ff0b3ea8</sub>